### ISSUE TITLE
Spintax File (like coinflip images)

### ISSUE THREAD

**Issue by KnightSec-Official** [vasani-arpit/WBOT#160]
**Is your feature request related to a problem? Please describe.**
Not really, idk, maybe

**Describe the solution you'd like**
"file": "{heads.png|tails.png}"

**Describe alternatives you've considered**
-

**Additional context**
I would like a file spintax, one where a random file will be selected and sent when user replies with an exact match...
i tried modifying some code in js but im trash and shit at js...

Any help will be appreciated!

Example of my idea:

```
         {
            "contains": [],
            "exact": ["!coinflip"],
            "response": "",
            "file": "{heads.png|tails.png}"
            }

**Comment by ryahconstantino** [comment]
I did something like that and it worked perfectly for me. Take a look at the code: 
https://github.com/vasani-arpit/WBOT/pull/65#issuecomment-583213778

Here's the spintax working with images:
![Screen Shot 2020-10-03 at 03 04 42](https://user-images.githubusercontent.com/17382946/94984712-5dc43280-0525-11eb-9f7d-296b5307fe18.png)

Now it's merged and working at version v0.14.0: https://github.com/vasani-arpit/WBOT/commit/078513bc91add27c89f485b95d7274e4448136cd

**Comment by KnightSec-Official** [comment]
THank you! <3  CLOSED
